external/jeigen/.gitignore

```diff
@@ -0,0 +1 @@
+/.gradle/
```

Comment:
Doesn't git only use the .gitignore at the project's root? Plus it would make more sense to modify that one for all submodules